Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
             targets: ["RxExtensions"])
    ],
    dependencies: [
-        .package(url: "https://github.com/ReactiveX/RxSwift.git", from: "5.0.0")
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", from: "5.1.1")
```

Comment:
Do we want to remove RxSwift as a dependency?

Reply:
I would do it for 5.0.0 with iOS 9 and 10 dropped (the other PR), but keep it in for 4.3.3. If you could merge this, merge other PRs and release it, we could then proceed with the drop and new 5.0.0 without Rx. :)